scripts/start.sh

```
@@ -0,0 +1,86 @@
+stopAny
+
+cd "$(distributionDirOf "$EDITION")"
+
+SH_FILE_DIR="khulnasoft-*/bin/$OS_DIR/"
+if ! ls $SH_FILE_DIR &> /dev/null; then
+  BASE_FILE_NAME="$(baseFileNameOf "$EDITION")"
+  echo "Unpacking ${BASE_FILE_NAME}..."
+  ZIP_FILE="${BASE_FILE_NAME}-*.zip"
+  unzip -qq ${ZIP_FILE}
+fi
+cd $(find khulnasoft-* -type d | head -1)
```

Shellcheck (0.11.0)

[warning] Quote this to prevent word splitting.

(SC2046)